wp-admin/options-web-app-manifest-display.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Web App manifest display field for the reading settings administration panel.
+ *
+ * As part of a core merge, the code in this file would go inside wp-admin/options-reading.php
+ *
+ * @package PWA
+ */
+
+namespace PWA_WP;
+
+/**
+ * Register web app manifest display setting.
+ */
+function register_web_app_manifest_display_setting() {
+	register_setting(
+		'reading',
+		'web_app_manifest_display',
+		array(
+			'type'              => 'string',
+			'sanitize_callback' => 'sanitize_text_field',
+		)
+	);
```

Comment:
We should make sure this doesn't end up adding `web_app_manifest_display` as an autoloaded option, since it is only used in the web app manifest request.

I think this may mean adding a call to `add_option` before this call.

```
add_option( 'web_app_manifest_display', 'minimal-ui', '', false );
```

Although there may be a better place for it.

I realize this should also be done for the `short_name` field.

Reply:
And for the `offline_browsing` option.